### ISSUE TITLE
How do you pretrain the modality transfer network?

### ISSUE THREAD

**Issue by czvvd** [Hydrogenion/ViPC#2]
the loss function is CDloss of complete point cloud and reconstrusted point cloud?

**Comment by Hydrogenion** [comment]
CD loss + EMD loss are used to train reconstrusted and complete point cloud.

**Comment by czvvd** [comment]
Thanks a lot,but I wonder that what gpu do you use to train the model and how long do you need to train 100 epochs?Because the dataset is so big,it contains over 730k training sets.

**Comment by Hydrogenion** [comment]
Yeah, the dataset is so big, maybe you sample a small set for debug your network. 
BTW, I use 2*V100 (32GB) for trainging it.  It takes about one week.

**Comment by Hydrogenion** [comment]
Yeah, the dataset is so big, maybe you sample a small set for debugging your network.
BTW, I use 2*V100 (32GB) and 8*GTX1080Ti(11GB) for trainging it. It takes about one week.

**Comment by czvvd** [comment]
Thanks a lot for releasing code, but it seems that the utlis folder still needs to be updated.

**Comment by Hydrogenion** [comment]
the utlis has been already committed!

**Comment by duzhenjiang113** [comment]
Hi, could you please tell me how to pre-train the modality transfer network?  Are you using the entire network for training? Do you only update the network parameters of modality transfer during training?
